todoman/model.py

```diff
@@ -70,7 +87,21 @@ def __init__(self, todo=None, filename=None):
         if self.todo.get('dtstamp', None) is None:
             self.todo.add('dtstamp', datetime.utcnow())
 
+        self.safe = safe or new
         self.filename = filename or "{}.ics".format(self.todo.get('uid'))
+        self.mtime = mtime or datetime.now()
+
+    @staticmethod
```

Comment:
In principle this should be a `classmethod` such that this returns the correct subclass when `Todo` is subclassed.